go-controller/pkg/cluster/master.go

```diff
@@ -46,6 +46,9 @@ func (cluster *OvnClusterController) StartClusterMaster(masterNodeName string) e
 	}
 	// Add the masterSwitchNetwork to subrange so that it is counted as one already taken
 	subrange = append(subrange, masterSwitchNetwork)
+	// Add the service subnet to subrange so that it is counted as one already taken
+	subrange = append(subrange, cluster.ClusterServicesSubnet)
```

Comment:
This subrange is already initialized with just the pod cidr network. So this operation is likely going to yield nothing. 
Unless the service cidr and pod cidr overlaps - which is the check that you may want to put. If the two networks overlap then error out or at least a warning in caps.